sites/funmooc/src/backend/templates/richie/base.html

```diff
@@ -21,4 +21,11 @@
 
 {% block body_footer_title %}{% endblock body_footer_title %}
 
-{% block body_js %}{% endblock body_js %}
\ No newline at end of file
+{% block body_js %}{% endblock body_js %}
+
+{% block body_footer_brand %}
+{# Change the link to point to our institutional site #}
+<a href="{% trans "https://www.france-universite-numerique.fr/en/" %}">
+    <img src="{% static "richie/images/logo-alt.png" %}" alt="">
+</a>
+{% endblock body_footer_brand %}
```

Comment:
Should we also apply this on FUN Campus and FUN Corporate?

Reply:
Yes good idea!